visualization/README.md

```diff
@@ -1,22 +1,12 @@
-# CodeCharta
+# CodeCharta visualization
 
 [![Build Status](https://travis-ci.org/MaibornWolff/codecharta.svg?branch=master)](https://travis-ci.org/MaibornWolff/codecharta)
 
 > CodeCharta by [MaibornWolff](https://www.maibornwolff.de)
 
-## Jump to Section
```

Comment:
Why are all jumps removed? Only URL Parameters should be removed

Reply:
Because the file is short enough to check what's included. If you want them back I would at least remove the `back to top`-links

Reply:
Yes, the back to top links are overkill. But I thought the sectionjumps are nice to have :)